engine/src/test/java/com/datatorrent/stram/cli/ApexCliTest.java

```diff
@@ -103,8 +107,8 @@ public static void finished()
   public void testLaunchAppPackagePropertyPrecedence() throws Exception
   {
     // set launch command options
-    ApexCli.LaunchCommandLineInfo commandLineInfo = ApexCli
```

Comment:
Why are we making such changes and have an extra import static? Please restrict your changes to **only** those that are relevant to this feature.